src/docbuilder/mod.rs

```diff
@@ -56,40 +52,43 @@ impl DocBuilder {
         Ok(())
     }
 
-
     fn load_database_cache(&mut self) -> Result<()> {
         debug!("Loading database cache");
+
         use crate::db::connect_db;
         let conn = connect_db()?;
 
-        for row in &conn.query("SELECT name, version FROM crates, releases \
-                               WHERE crates.id = releases.crate_id",
-                   &[])
-            .unwrap() {
+        for row in &conn
+            .query(
+                "SELECT name, version FROM crates, releases \
+                 WHERE crates.id = releases.crate_id",
+                &[],
+            )
+            .unwrap()
```

Comment:
Can you return this error instead of unwrapping?

Reply:
I'm not sure what the error type would be, or can I just `?`?

Reply:
Yes, `?` should work. All the error types are `failure::Error`, which implements `From` for most errors.